config/sync/views.view.taxonomy_term_pages.yml

```diff
@@ -441,7 +441,7 @@ display:
           click_sort_column: value
           type: entity_title_heading
           settings:
-            tag: h3
+            tag: h2
```

Comment:
These have to stay `h3` because of the grouping on the term pages. the "Assistant Professor" is an `<h2>` so therefore the person name should be an `h3`. There's a ticket for why this was done in the previous release.
![image](https://user-images.githubusercontent.com/7185045/199797437-9c8b09d9-3ba4-41f9-b2f6-60b6e289f0bd.png)

Reply:
@pookmish I undid the h2. Thank you for catching that.